--- a/numpy/core/setup_common.py.orig	2019-01-08 11:27:18.508528536 +0100
+++ b/numpy/core/setup_common.py	2019-01-08 11:29:05.782787928 +0100
@@ -189,8 +189,5 @@
     ]
 C99_COMPLEX_FUNCS = [
-    "cabs", "cacos", "cacosh", "carg", "casin", "casinh", "catan",
-    "catanh", "ccos", "ccosh", "cexp", "cimag", "clog", "conj", "cpow",
-    "cproj", "creal", "csin", "csinh", "csqrt", "ctan", "ctanh"
     ]
 
 def fname2def(name):
